src/assets/src/components/meetingTables.tsx

```diff
@@ -122,8 +122,11 @@ function UnstartedMeetingEditor (props: UnstartedMeetingEditorProps) {
 function StartedMeetingEditor (props: MeetingEditorProps) {
     const attendee = props.meeting.attendees[0];
     const attendeeString = `${attendee.first_name} ${attendee.last_name}`;
-    const joinUrl = props.meeting.backend_metadata?.meeting_url;
-    const role = props.user.id === props.meeting.assignee!.id ? 'Host' : 'Guest';
+    const isHost = props.user.id === props.meeting.assignee!.id;
+    const joinUrl = isHost
+        ? props.meeting.backend_metadata?.host_meeting_url || props.meeting.backend_metadata?.meeting_url
```

Comment:
I was thinking about the edge case of supporting old pre-1.4 Meetings in the UI. In that case, you might be the host, and the metadata might not have a `host_meeting_url`, so if I only included the left side, `joinUrl` would be `undefined`. Since I included the `||`, it'll be `meeting_url`, which is available on pre-1.4 Meetings.

Reply:
Nice thinking!